### PR TITLE
test(e2e,playwright): reset-modal.tsx

### PR DESCRIPTION
**Pull request by anand-harsh**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

#51705

### CODE REVIEW

**Comment by anand-harsh** [comment]
I am getting a problem that the pathUrl for page.goto which I am using is not matching as the Reset window is a pop up window, Can anyone help me how to approach this?

image

File: e2e/reset-modal.spec.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  // ...
+}
+
+test.beforeAll(async () => {
+  await seed();
+});
+
+test.beforeEach(async ({ page }) => {
+  await page.goto(
+    '/learn/2022/responsive-web-design/learn-html-by-building-a-cat-photo-app/step-1'
```
Comment (naomi-lgbt) [inline e2e/reset-modal.spec.ts:19]:
```suggestion
    '/learn/2022/responsive-web-design/learn-html-by-building-a-cat-photo-app/step-2'
```

The first step has the project preview modal, which might be what's interfering with the tests here.

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

File: e2e/reset-modal.spec.ts (hunk 1 above)
Comment (anand-harsh) [reply e2e/reset-modal.spec.ts:19]: > The first step has the project preview modal, which might be what's interfering with the tests here.

its not working, showing some out of viewport in safari and chromium

Comment (naomi-lgbt) [reply e2e/reset-modal.spec.ts:19]: Have you tried updating the step URL as I've indicated above?

Comment (anand-harsh) [reply e2e/reset-modal.spec.ts:19]: Yes, but tests are failing in webkit and safari

**Comment by raisedadead** [comment]
Hi @anand-harsh 

Thanks for taking the time to create this PR. So, as you know, we went with another PR for this component. However, we would love to see more contributions from you in the future, including any improvements you might be interested in adding to the tests.

Please feel free to work on something else in a new PR.

Thank you, and happy contributing.

**Pull request closed without merging**